### ISSUE TITLE
Automate Downstream Tests

### ISSUE THREAD

**Issue by VenelinMartinov** [pulumi/pulumi-terraform-bridge#1721]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details
We should automate the downstream test reaping. This will allow more automation on top, like nightly bridge releases.

POC https://github.com/pulumi/pulumi-terraform-bridge/pull/1718

**Comment by t0yv0** [comment]
https://github.com/pulumi/pulumi-terraform-bridge/blob/master/scripts/downstream_checks.py is there

**Comment by VenelinMartinov** [comment]
Yeah, great suggestion! That script will allow us to use one worker for all downstream PRs instead of one worker per PR
